tests/test_taipy.py

```diff
@@ -497,3 +497,38 @@ def assert_result_parents_and_expected_parents(parents, expected_parents):
         expected_parents = {}
         parents = tp.get_parents(scenario.cycle)
         assert_result_parents_and_expected_parents(parents, expected_parents)
+
+    def test_get_cycles_scenarios(self):
+        scenario_cfg_1 = Config.configure_scenario(
+            "s1",
+            [],
+            Frequency.DAILY,
+        )
+        scenario_cfg_2 = Config.configure_scenario("s2", [], Frequency.WEEKLY)
+        scenario_cfg_3 = Config.configure_scenario("s3", [], Frequency.MONTHLY)
+        scenario_cfg_4 = Config.configure_scenario("s4", [], Frequency.YEARLY)
+        scenario_cfg_5 = Config.configure_scenario("s5", [], None)
+
+        scenario_1_1 = tp.create_scenario(scenario_cfg_1, datetime.datetime.now())
+        scenario_1_2 = tp.create_scenario(scenario_cfg_1, datetime.datetime.now())
```

Comment:
Can you also test with the following?
```suggestion
        scenario_1_2 = tp.create_scenario(scenario_cfg_1, datetime.datetime.now())
        scenario_1_3 = tp.create_scenario(scenario_cfg_1, datetime.datetime.now() + datetime.timedelta(days=1))
        scenario_1_4 = tp.create_scenario(scenario_cfg_1, datetime.datetime.now()  + datetime.timedelta(days=8))
        scenario_1_5 = tp.create_scenario(scenario_cfg_1, datetime.datetime.now() + datetime.timedelta(days=25))
```